zerver/openapi/zulip.yaml

```diff
@@ -19204,6 +19392,66 @@ components:
 
             New in Zulip 8.0 (feature level 191). Previously, groups
             could be mentioned if and only if they were not system groups.
+    Invite:
+      type: object
+      description: |
+        A dictionary containing details about an invitation.
+      additionalProperties: false
+      properties:
+        id:
+          type: integer
+          description: |
+            The unique ID of the invitee if `is_multiuse` is `false` (i.e. invitation
```

Comment:
`The unique ID of the invitation link object`? "invitee" here sounds like it's a user ID or something.

Reply:
I am not sure what would be the correct description here. Can you suggest something?

Reply:
I was suggesting changing the start of this sentence to "The unique ID of the invitation link object".

Reply:
Oh sorry about that. I overlooked that line. I have fixed it now.